packages/thread-context/README.md

```
@@ -0,0 +1,248 @@
+// Opt-in: destroy active bridges on SIGTERM/SIGINT and re-raise the signal
+// so the host process exits naturally. Pass { exit: true } to force
+// `process.exit(143 | 130)` after teardown instead.
+installShutdownHandlers();
+```
+
+Signal handlers are **not** registered at import time — `installShutdownHandlers()`
+must be called by the host if that behavior is desired.
+
+## Architecture
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)